packages/components/src/components/Passphrase/EnterOnTrezorButton.tsx

```diff
@@ -20,19 +19,25 @@ interface EnterOnTrezorButtonProps {
 }
 
 export const EnterOnTrezorButton = ({ submit, value, deviceModel }: EnterOnTrezorButtonProps) => {
+    const theme = useTheme();
+
     return (
         <Card
             paddingType="small"
             onClick={() => submit(value, true)}
             data-test="@passphrase/enter-on-device-button"
         >
-            <Row>
+            <Row gap={spacings.lg} alignItems="center" justifyContent="space-between">
                 {deviceModel && <Image alt="Trezor" image={`TREZOR_${deviceModel}`} height={34} />}
-                <FormattedMessage
-                    id="TR_ENTER_PASSPHRASE_ON_DEVICE"
-                    defaultMessage="Enter passphrase on Trezor"
-                />
-                <Icon icon="ARROW_RIGHT" />
+                <Flex>
+                    <Text variant="tertiary">
+                        <FormattedMessage
+                            id="TR_ENTER_PASSPHRASE_ON_DEVICE"
+                            defaultMessage="or enter on Trezor"
```

Comment:
I am not sure if we need this `defaultMessage` here at all. To me it seems as relict of the past.

Maybe @komret would know? Can we just delete all places and rely on default messages in `messages.ts`M

Reply:
`defaultMessage` is necessary here because `@trezor/components` package doesn't have access to the `Translation` component from `@trezor/suite` and doesn't see the strings in `messages.ts`. But I am not sure how exactly it works and @mroz22 doesn't remember either. Surely it can be improved.

Reply:
So is it safe to delete the whole `defaultMessage`?

Reply:
Thanks @komret got it. 

@jvaclavik no, we have to keep it here if I understand correctly. 

However, this seem that this component shall not be in component package  at all. Why its there? I see its used only in Suite. Lets move it there then. Or?